components/Micrio.vue

```diff
@@ -23,25 +23,47 @@ useHead({
 });
 
 const micrioRef = ref<HTMLMicrioElement>();
+const markerRef = ref<Models.ImageCultureData.Marker>();
+const isPopoverHidden = ref<Boolean>(false);
+const previousMarkerId = ref<String>();
+const tourSteps = ref<Array<string>>();
+
+const currentStepIndex = computed(() => {
+  if (!tourSteps.value || !markerRef.value) return;
+  return tourSteps.value.indexOf(markerRef.value.id);
+});
+
+const handleTouchMove = (e) => {
+  isPopoverHidden.value = true;
+  const micrio = micrioRef.value;
+  const currentlySelectedId = micrio?.state.$marker.id
+  if (!currentlySelectedId) return
+  const marker = document.getElementById(currentlySelectedId);
+  if (!marker) return
+  marker.parentElement?.classList.remove("opened")
+  }
 
 onMounted(() => {
   const element = document.querySelector("micr-io")!;
   micrioRef.value = document.querySelector("micr-io") as HTMLMicrioElement;
   const micrio = micrioRef.value;
 
   micrio.defaultSettings = {
-    noZoom: true,
-    hookDrag: false,
-    hookPinch: false,
     freeMove: false,
     _markers: {
       noTitles: true,
       autoStartTour: true,
-      zoomOutAfterClose: true,
     },
   };
 
   element.addEventListener("show", (e: any) => {
+    const markerTour = micrio.$current.$data.markerTours![0].steps;
+    const markersRef = document.querySelectorAll("button.micrio-marker");
```

Comment:
This is no longer a ref (which is good)
```suggestion
    const markers = document.querySelectorAll("button.micrio-marker");
```